pkgs/applications/networking/modem-manager-gui/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, buildEnv, pkgconfig, python3, fetchhg, gtk3, glib, gdbm, gtkspell3, itstool, libappindicator-gtk3, perldevelPackages }:
+
+stdenv.mkDerivation rec {
+  name = "modem-manager-gui";
+  version = "0.4.17";
+  rev = "v${version}";
+
+  src = fetchhg {
+    url = https://linuxonly@bitbucket.org/linuxonly/modem-manager-gui;
+    rev = "8bcf842";
+    sha256 = "04p08gzvi068bd3slbnaghajwzp4h5jacib640k5shngqimkl84p";
+  };
+
+  patches = [ ./remove-fhs-gdbm-check.patch ];
+
+  nativeBuildInputs = [
+    pkgconfig
+    python3
+    perldevelPackages.Po4a
+  ];
+
+  buildInputs = [
+    gtk3
+    glib
+    gdbm
+    gtkspell3
+
+    itstool
```

Comment:
`itstool` belongs to `nativeBuildInputs`

Reply:
fixed